### ISSUE TITLE
Cannot migrate Derby database in linux

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#2651]
ERROR 2013-07-02 16:40:29,651 [Thread-0] com.mirth.connect.server.Mirth: Error establishing connection to database, aborting startup. Failed to start database '/usr/local/mirthconnect_derby/mirthdb' with class loader java.net.URLClassLoader@765a16, see the next exception for details.
INFO  2013-07-02 16:40:29,652 [Thread-4] com.mirth.connect.server.Mirth: shutting down mirth due to normal request
Database shut down normally.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-2721
Reporter: wayneh
Created: 2013-07-02T16:42:52.000-0700

**Comment by rbeckman-nextgen** [comment]
This error occurred because of a problem with the server build. The server was not removing old jars from the /lib/donkey folder and since we updated the Derby jars recently, both the old and new jars were being included in the build, which caused this error on Linux.

The problem with the build was resolved in MIRTH-2801, which also resolves this issue.

Imported Comment. Original Details:
Author: brentm
Created: 2013-07-24T10:47:46.000-0700
